calculator.py

```diff
@@ -0,0 +1,22 @@
+"""
+This script function to calculate numerical values that performs
+addition, substraction, division and multiplication
+This function accepts user input ie the 2 numerical values
+and return the result value
+"""
+ 
+# perform the addition function
+def addition(value_1,value_2):
+    return value_1 + value_2
+
+# perform the subtraction function

```

Comment:
- Leaving a comment for function is good, but the most preferable way for function is to use the docstring. 
- The following code example is a good practice on how we can write a docstring.
```python
def say_hello(name):
  """
  Function will return a greeting string with "Hello" with parameter name. 

 Params:
 - name - a name to say hello

  Example:
  >> say_hello("Jack Sparrow")
  "Hello Jack Sparrow !"

  """
  return f"Hello {name} !"
```
- It has the following components:
  - Objective
  - Function return
  - Parameter explanation